settings.py

```diff
@@ -56,6 +56,14 @@ class Config:
     # HGNC
     GENES_HGNC = 'http://ftp.ebi.ac.uk/pub/databases/genenames/new/json/hgnc_complete_set.json'
 
+    # PROGENY
+    PROGENY_FILENAME = file_or_resource('progeny_normalVStumor_opentargets.txt')
+    PROGENY_EVIDENCE_FILENAME = HOME_DIR + '/progeny-20-05-2018.json'
```

Comment:
I guess that using the google buckets for internal evidence generation will help us avoid using 'HOME_DIR'?

Reply:
On CKs branch, the PheWAS file settings were also in this file. Now they live in their own file in [phewascat/constants.py](https://github.com/opentargets/evidence_datasource_parsers/blob/master/modules/phewascat/constants.py). It would be good to be consistent for all internal evidence types.

Reply:
Yes, but we should address the phewas stuff in a separate pull request.